py-polars/polars/datatypes.py

```diff
@@ -407,19 +425,18 @@ def __hash__(self) -> int:
         Boolean: pa.bool_(),
         Utf8: pa.large_utf8(),
         Date: pa.date32(),
-        # handle temporal types that require units
         Datetime: pa.timestamp("us"),
-        (Datetime, "ms"): pa.timestamp("ms"),
-        (Datetime, "us"): pa.timestamp("us"),
-        (Datetime, "ns"): pa.timestamp("ns"),
+        Datetime("ms"): pa.timestamp("ms"),
+        Datetime("us"): pa.timestamp("us"),
+        Datetime("ns"): pa.timestamp("ns"),
         Duration: pa.duration("us"),
-        (Duration, "ms"): pa.duration("ms"),
-        (Duration, "us"): pa.duration("us"),
-        (Duration, "ns"): pa.duration("ns"),
+        Duration("ms"): pa.duration("ms"),
+        Duration("us"): pa.duration("us"),
+        Duration("ns"): pa.duration("ns"),
         Time: pa.time64("us"),
-        (Time, "ms"): pa.time32("ms"),
-        (Time, "us"): pa.time64("us"),
-        (Time, "ns"): pa.time64("ns"),
+        # Time("ms"): pa.time32("ms"),
```

Comment:
Was this intended?

Reply:
Yes - we don't actually support timeunits for the `Time` type; I must have added them before in anticipation of such a development, but as `Time(tu)` can't be declared, it can't exist in the lookup dict. If support gets added int he future, can reinstate.